src/wp-includes/block-editor.php

```diff
@@ -211,10 +211,19 @@ function get_default_block_editor_settings() {
 		);
 	}
 
+	$allowed_mime_types = get_allowed_mime_types();
+
+	// Iterate through mime types and remove those unsupported by the image editor.
+	foreach ( $allowed_mime_types as $mime_type_extensions => $mime_type ) {
+		if ( ! wp_image_editor_supports( array( 'mime_type' => $mime_type ) ) ) {
```

Comment:
The issue with this approach is that it prevents users from uploading mp3s or other non-image types.  Instead, we should add similar explicit checks for file type support for webp, avif and heic, the same way we do for media - https://github.com/WordPress/wordpress-develop/blob/a48e1806f767c550cbb8b66d672507f589a7be9a/src/wp-includes/media.php#L4352-L4365